### PR TITLE
Update manually run demos

### PR DESCRIPTION
**Pull request by mudit2812**
* Updated `ensemble_multi_qpu.py` to run parallel code without `QNodeCollection` with `dask`.
* Updated an image generated by `adjoint_diff_benchmarking.py`.
* Fixed `tutorial_quantum_transfer_learning.py` to use `torch.hstack` to preserve trainability. Predictions should now be as accurate as before.

### CODE REVIEW

File: demonstrations/ensemble_multi_qpu.py [hunk 1]
```diff
@@ -245,19 +246,23 @@ def circuit1(params, x=None):
...
 
-
 def decision(softmax):
     return int(torch.argmax(softmax))
 
 
 def predict_point(params, x_point=None, parallel=True):
-    results = qnodes(params, x=x_point, parallel=parallel)
+    if parallel:
+        results = tuple(dask.delayed(q)(params, x=x_point) for q in qnodes)
```
Comment (timmysilv) [inline demonstrations/ensemble_multi_qpu.py:261]: i didn't see any notes when removing QNodeCollection, but I feel like the intention wasn't for users to have to call dask directly? would need to ask someone who knows better

Comment (josh146) [reply demonstrations/ensemble_multi_qpu.py:261]: Yep the intention was to show users, through demos, how to call Dask and use it with PennyLane :)

Comment (josh146) [reply demonstrations/ensemble_multi_qpu.py:261]: This is something we realized we should have done from the beginning -- it is quite simple, highlights how well PL works with libraries like Dask (as long as there are no race conditions 😬), and sets up users to use Dask where they need to.

Comment (timmysilv) [reply demonstrations/ensemble_multi_qpu.py:261]: ah, ignore me then! looks excellent

**Review by timmysilv — APPROVED** [review]

File: demonstrations/ensemble_multi_qpu.py (hunk 1 above)
Comment (mudit2812) [reply demonstrations/ensemble_multi_qpu.py:261]: @trbromley is the author. He might have some insight.

**Review by albi3ro — APPROVED** [review]

**Pull request merged**